app/src/main/java/com/marvelapp/presentation/activity/MainMenuActivity.kt

```diff
@@ -0,0 +1,50 @@
+package com.marvelapp.presentation.activity
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.lifecycleScope
+import com.marvelapp.databinding.ActivityMainMenuBinding
+import com.marvelapp.presentation.viewmodel.MainMenuViewModel
+import com.marvelapp.presentation.viewmodel.MainMenuViewModel.MainMenuStates.BUTTON_PRESSED
+import com.marvelapp.presentation.viewmodel.MainMenuViewModel.MainMenuStates.WAITING
+import kotlinx.coroutines.flow.collectLatest
+import javax.inject.Inject
+
+class MainMenuActivity : AppCompatActivity() {
```

Comment:
this should have the "AndroidEntryPoint" annotation

Reply:
Solved